README.md

```diff
@@ -61,90 +66,208 @@ SENTRY_DNS=8.8.8.8
 # COMMON_API_CREDS
 REACT_APP_API_URL=https://myApiServerUrl.com
 ```
+
 NB:
-* Ensure you use the prefix `REACT_APP_` to store all the variables that are needed in the React Application. Example: `REACT_APP_NAME=janedoe`
-* **DO NOT** commit any of the environment files to version control.
+
+- Ensure you use the prefix `REACT_APP_` to store all the variables that are needed in the React Application. Example: `REACT_APP_NAME=janedoe`
+- **DO NOT** commit any of the environment files to version control.
 
 ### Docker
 
 This application is built with Docker, you will need to have the following installed in order to use it:
-* `Makefile` - Use the link below to install https://makefiletutorial.com/.
-* `Docker` - https://www.docker.com/.
+
+- `Makefile` - Use the link below to install https://makefiletutorial.com/.
+- `Docker` - https://www.docker.com/.
+
+make sure you are in the root directory when running make commands
+
+### General
 
 1. Build Docker container and start service.
+   ```
+   make docker-build
+   ```
+2. Build Docker container in test environment
+   ```
+   make docker-build-test
+   ```
+3. Show logs from container processes.
+   ```
+   make show-logs
+   ```
+4. Stop container.
+
+   ```
+   make start
+   ```
+
+5. reset containers
+   ```
+   make reset
+   ```
+6. Pruning system images
+   ```
+   make prune
+   ```
+
+### Server commands
+
+4. Build Docker container and start service.
+   ```
+   make api-gem-setup
+   ```
```

Comment:
The `make api-gem-setup` command is used to perform gem installations on the local machine.